packages/back-end/src/integrations/BigQuery.ts

```diff
@@ -78,4 +79,32 @@ export default class BigQuery extends SqlIntegration {
   castUserDateCol(column: string): string {
     return `CAST(${column} as DATETIME)`;
   }
+
+  async getInformationSchema(projectId: string) {
+    const datasets = await this.runQuery(
+      `SELECT * FROM ${projectId}.INFORMATION_SCHEMA.SCHEMATA;`
+    );
+
+    const combinedResults: SchemaResults[] = [];
+
+    for (const dataset of datasets) {
+      const sql = `SELECT
+        table_name,
+        column_name,
+        data_type
+      FROM
+        ${projectId}.${dataset.schema_name}.INFORMATION_SCHEMA.COLUMNS
+      ORDER BY
+        table_name;`;
+      const results = await this.runQuery(sql);
+
+      //MKTODO: Is there a awit to make this more efficient?
```

Comment:
Open to suggestions on how to make this more performant as currently it's got a bigO of $n^2